test/integration/index_documents_test.rb

```diff
@@ -0,0 +1,71 @@
+require "integration_test_helper"
+require 'govuk_message_queue_consumer'
+require 'govuk_message_queue_consumer/test_helpers'
+require './lib/index_documents'
+
+class IndexDocumentsTest < IntegrationTest
+  include IndexDocumentsTestHelpers
+
+  def setup
+    stub_elasticsearch_settings
+    create_test_indexes
+    stub_tagging_lookup
+    stub_calls_for_index_documents_test
+  end
+
+  def teardown
+    clean_test_indexes
+  end
+
+  def test_topics_get_indexed_from_publishing_api
+    Elasticsearch::Amender.any_instance.expects(:amend)
```

Comment:
I think this test still needs to check a bit more; it just checks that an amend call was made - I'd like it to at least check that it's trying to amend the right document, and that it's adding the appropriate tags (because I think those are more likely things to go wrong than it failing to call `amend` at all.